src/main/java/seedu/address/logic/newcommands/MoveStudentToGroupCommand.java

```diff
@@ -33,18 +45,93 @@ public MoveStudentToGroupCommand(RelativePath source, RelativePath dest) {
         this.dest = dest;
     }
 
+    /**
+     * Gets the relative path to the source group from which the student will be moved.
+     *
+     * @return The relative path to the source group.
+     */
+    public RelativePath getRelativePathToSourceGroup() {
+        return this.source;
+    }
+
+    /**
+     * Gets the relative path to the destination group to which the student will be moved.
+     *
+     * @return The relative path to the destination group.
+     */
+    public RelativePath getRelativePathToDestinationGroup() {
+        return this.dest;
+    }
```

Comment:
What are these two getter for? If no specific usage, avoid creating getter method to ensure data encapsulation.